coopaname_custom/models/hr_employee.py

```diff
@@ -81,15 +101,19 @@ def create_user_portal_access(self):
             if user:
```

Comment:
at line 97 (I can't comment on the code not added...), we should tell in the error which employee has no email address.